### PR TITLE
Fixed wrong bootstrap 4 classes on Classic Theme

### PR DESCRIPTION
**Pull request by SebBareyre**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | fix wrong bootstrap 4 class on classic theme
| Type?         | improvement
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | -
| How to test?  | -

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/8892)

### CODE REVIEW

**Comment by SebBareyre** [comment]
Oups, I updated the description :)

**Comment by mickaelandrieu** [comment]
@SebBareyre the pull request template is required, thanks for your contribution but I have the feeling that all your commits are the same?

Indeed, your contribution is really interesting :)

**Review by mickaelandrieu — DISMISSED** [review]

**Comment by kpodemski** [comment]
from what i remember this change was introduced **in version newer than this one used by PrestaShop**

**Comment by mickaelandrieu** [comment]
this is why I'd like a review from @eternoendless, because he's really aware of bootstrap version we are using and what is introduced or not ^^ I'm not really good in frontend stuff

**Comment by mickaelandrieu** [comment]
> Oups, I updated the description :)

Don't forget to fill the table, we won't merge it without it :/

**Comment by SebBareyre** [comment]
I fix it

**Comment by mickaelandrieu** [comment]
@SebBareyre thanks I need a review from @eternoendless.

We won't forget this contribution don't worry :)

File: themes/classic/templates/catalog/_partials/sort-orders.tpl
Comment (eternoendless) [inline themes/classic/templates/catalog/_partials/sort-orders.tpl:26]: Missing `col-sm-12` after `{else}`

File: themes/classic/templates/checkout/_partials/cart-detailed-product-line.tpl
Comment (eternoendless) [inline themes/classic/templates/checkout/_partials/cart-detailed-product-line.tpl:119]: Missing `col-md-6`

**Review by eternoendless — CHANGES_REQUESTED** [review]
Nice find! I think you over-deleted some classes though. Can you please restore them?

File: themes/classic/templates/catalog/_partials/sort-orders.tpl
Comment (SebBareyre) [reply themes/classic/templates/catalog/_partials/sort-orders.tpl:26]: `col-sm-12` is useless because there is `col-12` which is also 100% width
cf the doc here : https://v4-alpha.getbootstrap.com/layout/grid/

File: themes/classic/templates/checkout/_partials/cart-detailed-product-line.tpl
Comment (SebBareyre) [reply themes/classic/templates/checkout/_partials/cart-detailed-product-line.tpl:119]: here is the same, `col-md-6` is useless which is also 50% width
cf the doc here : https://v4-alpha.getbootstrap.com/layout/grid/

**Comment by SebBareyre** [comment]
@eternoendless : I answered you :)

**Review by SebBareyre — COMMENTED** [review]
`col-sm-12` is useless because there is `col-12` which is also 100% width
cf the doc here : https://v4-alpha.getbootstrap.com/layout/grid/

**Review by eternoendless — DISMISSED** [review]

**Comment by eternoendless** [comment]
Dammit, I should have paid more attention before reviewing.

> from what i remember this change was introduced in version newer than this one used by PrestaShop

@kpodemski is absolutely right. Flexbox was introduced in Bootstrap 4 alpha 6, which is why the Classic theme got stuck with Bootstrap 4 alpha 5. We cannot upgrade Bootstrap in 1.7 because it would break its layout and all of its children themes too.

Thanks for your contribution @SebBareyre but unfortunately we cannot accept it.

You can find an alternate version of Classic (which uses Bootstrap 4.1) here: https://github.com/PrestaShop/classic-rocket

**Pull request closed without merging**
